server/lib/sync-agent.js

```diff
@@ -386,6 +393,7 @@ export default class SyncAgent {
       last_updated_at,
       import_start_date: moment().subtract(options.import_days, "days").format()
     };
+    query = removeComments(query, replacements, this.hull);
```

Comment:
this is so that if we detect a variable in the comments it will throw an error right?

Reply:
Yes, and with some debugging logs on top of that.
There's another place where this function is called. It's in the `runQuery` function, I had to wrap it in a try catch otherwise the error wouldn't bubble up to the preview editor.